gmusicapi/clients/mobileclient.py

```diff
@@ -43,6 +43,14 @@ def _ensure_device_id(self, device_id=None):
 
         return device_id
 
+    def _validate_device_id(self, device_id):
+        """Ensure that a given device_id belongs to the user supplying it."""
+        if device_id in [d['id'][2:] if d['id'].startswith('0x') else d['id']
+                         for d in self.get_registered_devices()]:
+            return device_id
+        else:
+            raise ValueError('Invalid device_id.')
```

Comment:
I have a script for that [here](https://github.com/christopher-dG/pmcli/blob/master/script/get_dev_id.py), it only lists iOS and Android ids because I got a 403 with the "DESKTOP_APP", so I don't think people should be using those. Besides, I think it's safe to assume that the vast majority of people using this have an Android or iOS device. 

Re documentation: The people running into this are users rather than developers, so the extra information should be in output, rather than the docstring (or both). Maybe it could prompt the user asking if they want to see a list of their device ids, and then display them.